docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,258 @@
+* Puhti
+    * [List of all available datasets in Puhti](../../../data/datasets/spatial-data-in-csc-computing-env.md)
+    * Sentinel and Landsat mosaics of Finland provided by FMI and SYKE: ```/appl/data/geo/sentinel/s2```
+    * Accessing data in Puhti requires CSC user account with a project where Puhti service is enabled. All Puhti users have **read** access to data stored on Puhti, no need to move it, unless you need to modify it.
+* Allas
+    * [List of all available geospatial datasets in Allas](../../../data/datasets/spatial-data-in-csc-computing-env.md#spatial-data-in-allas)
+    * Sentinel-2 L2A data of Finland, where crops are grown, during growing seasons of 2016-present, [usage instructions](https://a3s.fi/sentinel-readme/README.txt)
+    * [Data from Allas can be read directly with different tools](../../../apps/gdal.md#using-files-directly-from-allas).
+        
+    
+**[Amazon Web Service (AWS) open EO data](https://registry.opendata.aws/?search=tags:gis,earth%20observation,events,mapping,meteorological,environmental,transportation)** provides many different worldwide datasets, including Landsat and Sentinel. Some of the data can be downloaded on "requestor pays" basis. The situation is changing all the time, currently [Sentinel-2 L2A Cloud-optimized Geotiffs](https://registry.opendata.aws/sentinel-2-l2a-cogs/) are available for free. AWS also offers the possibility for virtual machines, with both CPU and GPU.
+
+**[Data and Information Access Services (DIAS)](https://www.copernicus.eu/en/access-data/dias)** are paid processing platforms with direct data access. The services provide primarily Copernicus data.
+    
+**[Microsoft planetary computer](https://planetarycomputer.microsoft.com)** provides [worldwide EO data](https://planetarycomputer.microsoft.com/catalog). For [computing](https://planetarycomputer.microsoft.com/docs/concepts/computing/) it offers JupyterHub together with Dask Gateway, both CPUs and GPUs are avaialable. It is currently available in preview: [request access to the planetary computer](https://planetarycomputer.microsoft.com/account/request).
+
+**[Google Earth Engine](https://earthengine.google.com/)** is a processing platform, which requires [registration](https://signup.earthengine.google.com/). It can be accessed via browser and has worldwide analysis ready data available of, among others [Sentinel-2](https://developers.google.com/earth-engine/datasets/catalog/sentinel-2/) and [Sentinel-1](https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S1_GRD). In general JavaScript is used to use the platform, but also [Python](https://developers.google.com/earth-engine/guides/python_install) and [R](https://github.com/r-spatial/rgee) support exists. Check out [GEE's tutorials](https://developers.google.com/earth-engine/tutorials). Note that Google Cloud Storage might be needed to export large datasets.
```

Comment:
```suggestion
**[Google Earth Engine](https://earthengine.google.com/)** is a processing platform, which requires registration. It can be accessed via browser and has worldwide analysis ready data available of, among others <a href="https://developers.google.com/earth-engine/datasets/catalog/sentinel-2/" aria-label="Sentinel-2 data on GEE">Sentinel-2</a> and <a href="https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S1_GRD" aria-label="Sentinel-1 data on GEE">Sentinel-1 data</a> . In general, JavaScript is used on the platform, but also  <a href="https://developers.google.com/earth-engine/guides/python_install" aria-label="Python on GEE">Python</a>  and  <a href="https://github.com/r-spatial/rgee" aria-label="R on GEE">R</a> [GEE R]() support exists. Check out [GEE's tutorials](https://developers.google.com/earth-engine/tutorials). Note that Google Cloud Storage might be needed to export large datasets.
```